### PR TITLE
Support previewing rosbag files without rosbridge

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
Add feature to open a rosbag file directly by streaming from api-file-provider.

## Why?
Some rosbag files do not require additional ROS-nodes to preview the file.  
In that case, streaming data directly from the file improves user experience.

## Screenshot or video [Optional]
https://user-images.githubusercontent.com/24401842/140878120-37cb3c3e-256a-480a-8947-63cea21e6ecd.mov

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/216#" title="Author self-approved">d-hayashi</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/components/molecules/FilePreviewer/RosbagPreviewer/index.tsx
Comment (WatanabeToshimitsu) [inline src/components/molecules/FilePreviewer/RosbagPreviewer/index.tsx:64]: [variant="contained" はテーマの方で指定してる](https://github.com/dataware-tools/app-common/blob/master/src/theme.ts#L65)ので不要です！

**Review by WatanabeToshimitsu — CHANGES_REQUESTED** [review]
ほぼ OK です！

File: src/components/molecules/FilePreviewer/RosbagPreviewer/index.tsx
Comment (d-hayashi) [reply src/components/molecules/FilePreviewer/RosbagPreviewer/index.tsx]: なるほど、あざます！

**Pull request merged**
